web/src/components/storage/VolumeFields.jsx

```diff
@@ -157,9 +127,10 @@ const FsSelectOption = ({ fsOption }) => {
  * @param {string} props.id - Widget id.
  * @param {string} props.value - Currently selected file system.
  * @param {Volume} props.volume - The selected storage volume.
- * @param {onChangeFn} props.onChange - Callback for notifying input changes.
+ * @param {boolean} props.isDisabled
+ * @param {(data: object) => void} props.onChange - Callback for notifying input changes.
  */
-const FsSelect = ({ id, value, volume, onChange }) => {
+const FsSelect = ({ id, value, volume, isDisabled, onChange }) => {
```

Comment:
Question: below, in the line 154 (former 174), the onChange callback sent `{ fsType: option, snapshots: false }`. Still `snapshots: false` being relevant here?

Reply:
I good say it is even wrong. Snapshots should be configured only by means of the snapshots field. Good catch!